conda_smithy/cirun_utils.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+from cirun import Cirun
+
+CIRUN_INSTALLATION_ID = os.environ.get("CIRUN_INSTALLATION_ID", 18453316)
```

Comment:
Add a comment about how to obtain this magic number, and how this is conda-forge's.